lib/formSchemaGenerator.ts

```
@@ -0,0 +1,261 @@
+              field.maxLength || 1000,
+              field.errorMessage ||
+                `Maximum ${field.maxLength || 1000} characters allowed`
+            );
+          break;
+
+        case 'select':
+          const options =
+            field.selectOptions
+              ?.split(',')
+              .map((option) => option.trim())
+              .filter(Boolean) || [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)